docs/rules/no-triple-slash-reference.md

```diff
@@ -0,0 +1,31 @@
+# Enforces reference comments are not used (no-triple-slash-reference)
+
+Reference comments, `/// <reference />`, should not be used anymore. Use `import` instead.
```

Comment:
Can you explain this a bit more? As someone unfamiliar with TypeScript, I think more context would be helpful.